docs/stdlib/sequence.rst

```diff
@@ -37,21 +37,21 @@ Sequences
             }
         }
 
-    The sequence is bound to the scalar type, not to the property, so
-    if multiple properties use the same ``sequence`` type they will
-    share the same counter. For each distinct counter, a separate
-    scalar type that is extending ``sequence`` should be used.
+    A sequence is bound to the scalar type, not to the property, so
+    if multiple properties use the same sequence, they will share the same
+    counter. For each distinct counter, a separate scalar type that is
+    extending :eql:type:`sequence` should be used.
 
 
 ---------
 
 
 .. eql:function:: std::sequence_next(seq: schema::ScalarType) -> int64
 
-    Advance the sequence to its next value and return that value.
+    Returns the next incrementation of a sequence to its next value.
 
-    Sequence advancement is done atomically, each concurrent session and
-    transaction will receive a distinct sequence value.
+    Sequence advancement is done atomically; each concurrent session and
+    transaction will receive a distinct sequence value:
```

Comment:
```suggestion
    transaction will receive a distinct sequence value.
```
The code block doesn't illustrate this idea in particular, unless I'm misinterpreting.